src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -982,8 +983,19 @@ pub const _SC_V7_LPBIG_OFFBIG: ::c_int = 125;
 pub const _SC_THREAD_ROBUST_PRIO_INHERIT: ::c_int = 126;
 pub const _SC_THREAD_ROBUST_PRIO_PROTECT: ::c_int = 127;
 
-pub const WCONTINUED: ::c_int = 4;
-pub const WSTOPPED: ::c_int = 0o177;
+pub const WNOHANG: ::c_int = 0x1;
+pub const WUNTRACED: ::c_int = 0x2;
```

Comment:
We already have them with the same values here: https://github.com/rust-lang/libc/blob/5786455a7900ad1a502dcf848e1e3ec77d669e95/src/unix/linux_like/mod.rs#L1085-L1086
So you don't have to re-define them.